[MTM-63337] Release notes for Client-Suggested Identity Provider for … (#3793)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-2025-261-0-Client-suggested-identity-provider.md

---

> date: '2025-10-10'

Reviewer: ```suggestion
date: 
```

Reviewer: Please leave the date always empty as it will be added automatically.

> title: Client-Suggested Identity Provider for SSO Login

Reviewer: ```suggestion
title: Client-suggested identity provider for SSO login
```

change_type:
  - value: change-QHu1GdukP
    label: Feature
product_area: Platform services
component:
  - value: q3kclF6pO
    label: Authentication
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-63337
version: 2025.261.0
---

> Added support for the idp_hint parameter to improve SSO login with multiple identity providers.

Reviewer: ```suggestion
To improve the single sign-on (SSO) login with multiple identity providers, a new `idp_hint` parameter has been added. 
```
